src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBanner15.vue

```diff
@@ -145,169 +82,70 @@ export default {
+			fifteenYearGoalPercent: 0
 		};
 	},
 	apollo: {
-		query: appealBannerQuery,
+		query: recentFundedLoans,
 		preFetch: true,
+		preFetchVariables() {
+			return {
+				start: '2020-10-01'
+			};
+		},
 		result({ data }) {
-			this.amountRaised = _get(data, 'general.kivaStats.latestDonationCampaign.amount_raised');
-			this.targetAmount = _get(data, 'general.kivaStats.latestDonationCampaign.target_amount');
+			const numRecentFundedLoans = data?.general?.kivaStats?.numRecentFundedLoans || 0;
+			if (numRecentFundedLoans !== 0) {
+				this.fifteenYearGoalPercent = Math.round(
+					(numRecentFundedLoans / 15000) * 100
+				);
+			}
 		},
 	},
 	computed: {
-		// GROW-230
-		// Checking to see if user is in experiment
-		// Boolean value determins a dynamic class being applied
-		// to be used for css overrides
-		isUserInExp() {
-			if (this.$route.name === 'homepage' && this.forceDismissOverlayExperiment.version === 'variant-b') {
-				return true;
+		bannerHeadline() {
+			const appealHeadline = this.appealBannerContent?.richText || '';
+			// contentful wraps all richText fields with a <p> tag,
+			// which makes them difficult to style as headers,
+			// this removes that wrapping tag
+			const options = {
+				renderNode: {
+					paragraph: (node, next) => `${next(node.content)}`
+				}
+			};
+			if (!this.open) {
+				return this.truncateStringToNumberOfWords(documentToHtmlString(appealHeadline, options), 5);
 			}
-			return false;
-		},
-		bannerHeadline(props) {
-			const appealHeadline = props?.appealBannerContent?.richText || '';
-			return documentToHtmlString(appealHeadline);
+			return documentToHtmlString(appealHeadline, options);
 		},
-		bannerBody(props) {
-			const appealBody = props?.appealBannerContent?.additionalContent?.[0].fields.richText || '';
+		bannerBody() {
+			const appealBody = this.appealBannerContent?.additionalContent?.[0].fields.richText || '';
 			return documentToHtmlString(appealBody);
 		},
-		buttonAmounts(props) {
-			const defaultAmounts = [20, 35, 50];
-			const donationAmounts = props?.appealBannerContent?.dataObject?.donationAmounts || [];
-			return donationAmounts && donationAmounts.length ? donationAmounts : defaultAmounts;
+		ctaButtonText() {
+			const buttonText = this.appealBannerContent?.dataObject?.ctaButtonText || '';
+			return buttonText;
+		},
+		shrinkButtonText() {
+			const buttonText = this.appealBannerContent?.dataObject?.shrinkButtonText || '';
+			return buttonText;
 		},
-		// NOT YET HOOKED UP
-		// bannerMatchingText(props) {
-		// 	let matchingText = props.appealBannerContent.dataObject.matchingTextReplacement;
-
-		// 	if (props.appealMatchEnabled === true) {
-		// 		// eslint-disable-next-line max-len
-		// 		matchingText = props.appealBannerContent.dataObject.matchingText;
-		// 	}
-		// 	return matchingText;
-		// },
 	},
 	mounted() {
-		if (store2.session.get('appeal_banner_shrunk')) {
+		if (store2.session.get('appeal_banner_15_shrunk')) {
```

Comment:
We need to ensure the full banner never renders as open after being closed for any reason. I initially though this would require moving to a cookie storage setting which could be read during ssr...but we can prob just move this up into created and fire in the client only...